setup.py

```diff
@@ -337,6 +342,7 @@ def do_setup():
             'async': async_packages,
             'azure_blob_storage': azure_blob_storage,
             'azure_data_lake': azure_data_lake,
+            'azure_kubernetes_service': azure_kubernetes_service,
```

Comment:
Do you really need a new group? Google Cloud Platform (`gcp_api`) does not create a separate group for each of your services

Reply:
I submit PR a few days ago. It was merged today. Look at: https://github.com/apache/airflow/pull/4524